File: backend/src/main/java/com/ll/hotel/domain/member/member/type/BusinessApiProperties.java
package com.ll.hotel.domain.member.member.type;

import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

@Component
@Getter
@Setter
@ConfigurationProperties(prefix = "business-api")
public class BusinessApiProperties {
    private String serviceKey;
    private String validationUrl;
    private String statusUrl;
}

File: backend/src/main/java/com/ll/hotel/domain/member/member/type/BusinessApiProperties.kt
package com.ll.hotel.domain.member.member.type

import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component

@Component
@ConfigurationProperties(prefix = "business-api")
class BusinessApiProperties {
    lateinit var serviceKey: String
    lateinit var validationUrl: String
    lateinit var statusUrl: String
}